lib/src/readers/content_reader.dart

```diff
@@ -6,6 +6,29 @@ import '../ref_entities/epub_content_ref.dart';
 import '../ref_entities/epub_text_content_file_ref.dart';
 import '../schema/opf/epub_manifest_item.dart';
 
+//Accented characters throw a Invalid argument(s): Illegal percent encoding in URI on Uri.decodeFull. 
+//By checking if it throws an error and returning the filename again if so, you can use accented chars in
+//epub internal filenames. 
+String accentedFileChecker(String incomingFileName) {
```

Comment:
this might be simpler:

String decodeFileName(String incomingFileName) {
  try {
    decodedFileName = Uri.decodeFull(incomingFileName);
  } catch (e) {
    decodedFileName = incomingFileName;
  }
  return decodedFileName;
}